util/git/creds.go

```diff
@@ -44,13 +44,7 @@ const (
 )
 
 func init() {
-	githubAppCredsExp := common.GithubAppCredsExpirationDuration
-	if exp := os.Getenv(common.EnvGithubAppCredsExpirationDuration); exp != "" {
-		if qps, err := strconv.Atoi(exp); err != nil {
-			githubAppCredsExp = time.Duration(qps) * time.Minute
-		}
-	}
-
+	githubAppCredsExp := env.ParseDurationFromEnv(common.EnvGithubAppCredsExpirationDuration, common.GithubAppCredsExpirationDuration, 1*time.Hour, 24*time.Hour)
```

Comment:
I think this is a breaking change. Prior to this, the expiration was given in minutes, without any duration suffix. This change makes the unit a duration. So, people having specified a given amount of minutes (e.g. 100) would suddenly see much shorter period of expiration (I believe the unit of Duration without modifier is nanoseconds).

Also, the minimum and maximum allowed values haven been enforced prior to this. If they are enforced, the minimum and maximum should be documented.  May I ask the reason for those value?